man/systemd.network.xml

```diff
@@ -2457,6 +2457,26 @@
         </listitem>
       </varlistentry>
 
+       <varlistentry>
+        <term><varname>CakeOverhead=</varname></term>
+        <listitem>
+          <para>Specifies the size of each packet on the wire may differ from that seen by Linux. Takes one of
+          <literal>ethernet</literal>, <literal>ether-vlan</literal> or ranges a number between -64 and 256.
```

Comment:
"or ranges a number" → "or a number in the range -64 … 256"

Reply:
hmm, why abbreviate "ethernet" in "ether-vlan" but not in "ethernet"? I'd not abbreviate either, i.e. use "ethernet" and "ethernet-vlan" as identifiers.